assignments/arrays.js

```diff
@@ -75,12 +75,13 @@ let inventory = [
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*`);
+console.log(`Car 33 is a *2011* *jeep* *wrangler*`);
```

Comment:
you're cheating here also. again, pretend you couldn't see the contents of the array. That is, you should access the array by the index. Something like...

`console.log("Car 33 is a " + inventory[i].car_year + " ")`

The reason why you should always do this is that the contents of the array might change. So if you access it by the index you will always have the right value.